Resources/Locale/en-US/components/window-component.ftl

```diff
@@ -0,0 +1,14 @@
+### UI
+
+# Shown when examining the window. Each entry represents the window's health condition
+window-component-damaged-1 = It looks fully intact.
+window-component-damaged-2 = It has a few scratches
+window-component-damaged-3 = It has a few small cracks.
+window-component-damaged-4 = It has several big cracks running along its surface.
+window-component-damaged-5 = It has deep cracks across multiple layers.
+window-component-damaged-6 = It is extremely badly cracked and on the verge of shattering.
```

Comment:
"extremely badly cracked" is crap grammar IMO.
"Extremely cracked" would suffice.